pkg/daemon/ceph/client/pool.go

```diff
@@ -283,6 +283,20 @@ func setCommonPoolProperties(context *clusterd.Context, clusterInfo *ClusterInfo
 		}
 	}
 
+	// set max_bytes and max_objects quotas
+	if pool.Quotas.Enabled {
+		// set max_bytes quota, 0 value disables quota
+		err := setPoolQuota(context, clusterInfo, poolName, "max_bytes", strconv.FormatUint(pool.Quotas.MaxBytes, 10))
+		if err != nil {
+			return errors.Wrapf(err, "failed to set max_bytes quota for pool %q", poolName)
+		}
+		// set max_objects quota, 0 value disables quota
+		err = setPoolQuota(context, clusterInfo, poolName, "max_objects", strconv.FormatUint(pool.Quotas.MaxObjects, 10))
```

Comment:
These settings can be applied whether replica or EC pools, right?

Reply:
That is correct.